src/components/split-button/split-button.style.ts

```diff
@@ -5,17 +5,26 @@ import baseTheme from "../../style/themes/base";
 
 const StyledSplitButton = styled.div`
   ${margin}
-
   display: inline-block;
   position: relative;
 
+  > {
```

Comment:
You'll need to cover for scenario where there's only one child

![image](https://user-images.githubusercontent.com/44157880/229561829-086d6a93-ebaa-4127-b12f-5093db7cab89.png)

Reply:
![image](https://user-images.githubusercontent.com/44157880/229568840-230ab079-b213-4d57-a060-e43ca94d2ade.png)

The below is pone way we could do it

```
 > {
      &:first-child:last-child {
        border-radius: var(--borderRadius100);
      }
      &:first-child:not(:last-child) {
        border-top-left-radius: var(--borderRadius100);
        border-top-right-radius: var(--borderRadius100);
        border-bottom-right-radius: var(--borderRadius000);
        border-bottom-left-radius: var(--borderRadius000);
      }

      &:not(:first-child):not(:last-child) {
        border-radius: var(--borderRadius000);
      }

      &:last-child:not(:first-child) {
        border-top-right-radius: var(--borderRadius000);
        border-top-left-radius: var(--borderRadius000);
        border-bottom-left-radius: var(--borderRadius100);
        border-bottom-right-radius: var(--borderRadius100);
      }
    }
```